DQMOffline/Trigger/python/DiJetMonitor_cff.py

```diff
@@ -0,0 +1,162 @@
+DiPFjetAve80_HFJEC_Prommonitoring = DiPFjetAve40_Prommonitoring.clone()
+DiPFjetAve80_HFJEC_Prommonitoring.FolderName = cms.string('HLT/JetMET/HLT_DiPFJetAve80_HFJEC/')
+DiPFjetAve80_HFJEC_Prommonitoring.histoPSet.dijetPtThrPSet = cms.PSet(
+  nbins = cms.uint32 (  100 ),
+  xmin  = cms.double(   0.),
+  xmax  = cms.double(  200.),
+)
+DiPFjetAve80_HFJEC_Prommonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DiPFJetAve80_HFJEC_v*")
+
+# HLT_DiPFJetAve100_HFJEC
+DiPFjetAve100_HFJEC_Prommonitoring = DiPFjetAve40_Prommonitoring.clone()
+DiPFjetAve100_HFJEC_Prommonitoring.FolderName = cms.string('HLT/JetMET/HLT_DiPFJetAve100_HFJEC/')
+DiPFjetAve100_HFJEC_Prommonitoring.histoPSet.dijetPtThrPSet = cms.PSet(
+  nbins = cms.uint32 (  50 ),
+  xmin  = cms.double(   0.),
+  xmax  = cms.double(  250.),
+)
+DiPFjetAve100_HFJEC_Prommonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DiPFJetAve100_HFJEC_v*")
+
+# HLT_DiPFJetAve160_HFJEC
+DiPFjetAve160_HFJEC_Prommonitoring = DiPFjetAve40_Prommonitoring.clone()
+DiPFjetAve160_HFJEC_Prommonitoring.FolderName = cms.string('HLT/JetMET/HLT_DiPFJetAve160_HFJEC/')
+DiPFjetAve160_HFJEC_Prommonitoring.histoPSet.dijetPtThrPSet = cms.PSet(
+  nbins = cms.uint32 (  80 ),
+  xmin  = cms.double(   0.),
+  xmax  = cms.double(  400.),
+)
+DiPFjetAve160_HFJEC_Prommonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DiPFJetAve160_HFJEC_v*")
+
+# HLT_DiPFJetAve220_HFJEC
+DiPFjetAve220_HFJEC_Prommonitoring = DiPFjetAve40_Prommonitoring.clone()
+DiPFjetAve220_HFJEC_Prommonitoring.FolderName = cms.string('HLT/JetMET/HLT_DiPFJetAve220_HFJEC/')
+DiPFjetAve220_HFJEC_Prommonitoring.histoPSet.dijetPtThrPSet = cms.PSet(
+  nbins = cms.uint32 (  55 ),
+  xmin  = cms.double(   0.),
+  xmax  = cms.double(  550.),
+)
+DiPFjetAve220_HFJEC_Prommonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DiPFJetAve220_HFJEC_v*")
+
+# HLT_DiPFJetAve300_HFJEC
+DiPFjetAve300_HFJEC_Prommonitoring = DiPFjetAve40_Prommonitoring.clone()
+DiPFjetAve300_HFJEC_Prommonitoring.FolderName = cms.string('HLT/JetMET/HLT_DiPFJetAve300_HFJEC/')
+DiPFjetAve300_HFJEC_Prommonitoring.histoPSet.dijetPtThrPSet = cms.PSet(
+  nbins = cms.uint32 (  75 ),
+  xmin  = cms.double(   0.),
+  xmax  = cms.double(  750.),
+)
+DiPFjetAve300_HFJEC_Prommonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DiPFJetAve300_HFJEC_v*")
+
+HLTDiJetmonitoring = cms.Sequence(
+    DiPFjetAve40_Prommonitoring
+    *DiPFjetAve60_Prommonitoring
+    *DiPFjetAve80_Prommonitoring
+    *DiPFjetAve140_Prommonitoring
+    *DiPFjetAve200_Prommonitoring
+    *DiPFjetAve260_Prommonitoring
+    *DiPFjetAve320_Prommonitoring
+    *DiPFjetAve400_Prommonitoring
+    *DiPFjetAve500_Prommonitoring
+    *DiPFjetAve60_HFJEC_Prommonitoring
+    *DiPFjetAve80_HFJEC_Prommonitoring
+    *DiPFjetAve100_HFJEC_Prommonitoring
+    *DiPFjetAve160_HFJEC_Prommonitoring
+    *DiPFjetAve220_HFJEC_Prommonitoring
+    *DiPFjetAve300_HFJEC_Prommonitoring
```

Comment:
seems excessive.. how many histogram bins are there per module?

Reply:
Each histogram bins of jetpt1, pt2, ptAvgB, ptAvgA, ptTag, ptProb is 200.
Each histogram bins of etaProb and etaTag is 50.
Each histogram bins of  ptAsym is 100.
Each histogram bins of ptAsymVsetaProb, etaPrbVSphiPrb and jetPhiPrb are 150, 114, 64, respectively.
The histogram bins of jetptThr is almost half of trigger threshold.
Per module (trigger path), histogram bins is normally 1800.
If I was wrong, please let me know.
Thank you.

Reply:
About total histogram bins is 27070.
Thank you.:)